share/revng/test/configuration/revng-qa/compile.yml

```diff
@@ -1,11 +1,12 @@
 tags:
   - name: stripped
   - name: with-debug-info
+  - name: has-special-compilation-requirements
```

Comment:
This really looks like a hack. Can you detail so we can try to get around it?

Reply:
I want to reuse our usual architecture tags (`i386`, `arm`, you get the point), but as soon as something is tagged with them - it gets compiled normally. So it was either a separate set of abi-specific tags (🤢) or something like this to prevent select sources from being sweeped.

Reply:
Yes, exactly.

Reply:
Ping.

Reply:
I thought about it. Can you introduce a tag that has the opposite meaning of `has-special-compilation-requirements`?
Like `simple-executable`. We can attach to all the sources that do not have `has-special-compilation-requirements` and have `revng-qa.compiled` to only use `simple-executable`.

Basically `revng-qa.compiled` would become "compile regular, simple, executable".

Reply:
Done.